src/constants/transformation/transform-service.ts

```
@@ -40,8 +40,9 @@ export class TransformService {
         if (!data.customFields || !Array.isArray(data.customFields)) {
           return null;
         }
-        const field = data?.customFields.find((f: any) => f?.label === label);
-
+        const field = data?.customFields.find((f: any) => {
+          return f?.label === label
```

ESLint

[error] Insert `;`

(prettier/prettier)